netci.groovy

```diff
@@ -30,6 +30,9 @@ Constants.scenarios.each { scenario ->
                 if ((configuration == 'Release' || os != 'Windows_NT') && scenario == 'corefx') {
                     return
                 }
+                if (os != 'Windows_NT' && scenario=='wasm') {
```

Comment:
Nit: could could be consistent about spaces between operators (and `if(` elsewhere).

Reply:
Fixed in the new version